internal/yggdrasil/yggdrasil.go

```diff
@@ -55,6 +62,24 @@ func NewYggdrasilHandler(deviceRepository edgedevice.Repository, deploymentRepos
 	}
 }
 
+func isRegisteredURL(url *url.URL) bool {
```

Comment:
`isRegistrationURL`?

Reply:
Keep here until ECOPROJECT-401 check this again.

Reply:
I was suggesting changing the name of the method to `isRegistrationURL`; after all it's not a "registered URL".